docs/using-radicle/identity.md

```diff
@@ -0,0 +1,80 @@
+---
+id: identity
+title: Manage your Radicle identity
+---
+
+Your Radicle identity is your ticket to hosting and collaborating on code in the Radicle network, but it's not a fixed
+entity. You can manage multiple identities, creating/delete them at will, and even edit them to a degree.
+
+Many of the actions below require your personal URN, which you can find using `rad self`.
+
+## View your active identity
+
+Run `rad self` to see details on how you're identified on the Radicle network, including your:
+
+- Active [display name](#edit-your-display-name)
+- Personal URN
+- Peer ID
+- Hashed and full encryption keys
+- File locations of your Git monorepo and encryption keys
+
+## Create a new identity
+
+Add new identities at any time. For example, use different identities for the personal projects you host on Radicle
+versus your professional work for a company or DAO.
+
+```
+rad auth --init
+```
+
+## Switch between identities
+
+If you have multiple identities, switch between them with `rad auth`.
+
+Choose the identity you want active, enter your passphrase, and hit **Enter**.
+
+## Edit your display name
+
+Change your _display name_, which is how you're identified on the Radicle web app (not the same as your Peer ID or
+personal URN!):
+
+```
+rad edit
+```
+
+Change the `"name"` value to your new display name in the file that appears.
+
+```
+{
+  "https://radicle.xyz/link/identities/person/v1": {
+    "name": "your-display-name"
+  }
+}
+```
+
+Re-run `rad self` to see that your **Name** changed, but your personal URN and Peer ID stayed the same.
+
+## Remove an identity
+
+Remove any identity from your local machine with the following:
+
+```
+rad rm YOUR-PERSONAL-URN
```

Comment:
Perhaps tangentially related to the previous issue, but the removal isn't working for me.

My workflow is as follows:

1. Check current profile/identity

```
(base) bordumb@bordumbs-MacBook-Pro ~ % rad self
Profile b172f291-f9ab-4990-b94e-738796026f50
├── Name           testBordumb                                                                                                   
├── URN            rad:git:hnrkdezzqcy45o487damow6ibwa8kwuodi6bo                                                                 
├── Peer ID        hybg1qak3g5cjgmsxh1oztejsytgwzd9n4z1wqju4eamcfu6w6oxus                                                        
├── Key (hash)     SHA256:A9sS2r0eK5lAj/tRqZ34wJB4T5ZrwpzcWGQy2Mq+318                                                            
├── Key (full)     ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIE0nYVk22JMuz+SheKE2BE1Lj+LV5UcmekYWws/U9B87                              
├── Storage (git)  /Users/bordumb/Library/Application Support/xyz.radicle.radicle-link/b172f291-f9ab-4990-b94e-738796026f50/git  
└── Storage (keys) /Users/bordumb/Library/Application Support/xyz.radicle.radicle-link/b172f291-f9ab-4990-b94e-738796026f50/keys 
```

2. Run removal using URN

```
(base) bordumb@bordumbs-MacBook-Pro ~ % rad rm rad:git:hnrkdezzqcy45o487damow6ibwa8kwuodi6bo
== Removing failed missing field `subject` at line 1 column 156
```

Any ideas on workaround here? Or if we're missing something?

Reply:
I get the same error, but I get errors on lots of the newer `rad` commands, so I figured it was on my end, not `rad` itself. 

I just put something on Discord to figure this out—I might have to remove this entire section in the meantime.

Reply:
Given what @cloudhead said in Discord, I'm reverting to manual deletion in this PR until there's a more formal way to delete an identity (with `rad rm` or some other function).